docs/user-guide/advanced/restore.md

```diff
@@ -0,0 +1,20 @@
+# Restoring from backup
+
+In order to fully restore your PhotoPrism instance you need the following:
+
+* All photo, video, and sidecar files in your `originals` folder
```

Comment:
sidecar files are not part of 'originals'. They are generated files and stored under 'storage'. I would not specify it here.

Reply:
There can in fact be original sidecar files, for example when you process a RAW with Photoshop and it creates a JPEG and XMP file for the original RAW. The dedicated sidecar folder is for sidecar files PhotoPrism creates so that they don't get mixed with manually created files.

Reply:
So, resolved?

Reply:
Yes :)